data-products/src/firefox_newtab/sql/ads_impressions_hourly/ads_daily_engagement_by_tile_id_position/data.sql

```diff
@@ -0,0 +1,27 @@
+SELECT
+  DATE(i.submission_timestamp) AS happened_at,
+  NULL AS recommendation_id,
+  COALESCE(SAFE_CAST(i.metrics.string.ad_id AS int), -1) AS tile_id,
```

Comment:
Let's also call this `ad_id`, we don't need to change the name here
```suggestion
  COALESCE(SAFE_CAST(i.metrics.string.ad_id AS int), -1) AS ad_id,
```